ci/run-k8s-e2e-tests.sh

```diff
@@ -139,22 +139,27 @@ echoerr "Using this sonobuoy: $SONOBUOY"
 function run_sonobuoy() {
     local focus_regex="$1"
     local skip_regex="$2"
+
+    docker pull projects.registry.vmware.com/antrea/sonobuoy-systemd-logs:v0.3
+    docker tag projects.registry.vmware.com/antrea/sonobuoy-systemd-logs:v0.3 sonobuoy/systemd-logs:v0.3
```

Comment:
Is it useful to pull this image here? I thought `sonobuoy/systemd-logs` was used to create a DaemonSet in the cluster under test. In which case, each Node needs to be able to pull the image, but pulling the image on the machine running the `sonobuoy` binary does not serve a real purpose.

Reply:
You are right. Moved this to `test.sh`.